### ISSUE TITLE
Lastest Version of Server won't load - Crashes

### ISSUE THREAD

**Issue by Pepster3** [YTG1234/manhunt-fabric#7]
**Describe the bug**
Server crashes at startup, only Mod files are Fabric API and Manhunt 2.0.2 - each downloaded today to ensure latest versions. 

**To Reproduce**
Create a Fabric 1.16.5 server 
Upload Fabric API and Manhunt JAR files to MOD folder
Start Server

**Expected behaviour**
Server should start without crashing

**Server Console Log**
```
21.02 16:15:56 [Multicraft] Loading server properties
21.02 16:15:56 [Multicraft] Starting server!
21.02 16:15:56 [Multicraft] Loaded config for "Fabric"
21.02 16:15:56 [Multicraft] Updating eula.txt file
21.02 16:15:57 [Server] main/INFO Loading for game Minecraft 1.16.5
21.02 16:15:57 [Server] ForkJoinPool-1-worker-9/WARN The mod "autoconfig1u" contains invalid entries in its mod json:
21.02 16:15:57 [Server] INFO - Unsupported root entry "$schema" at line 2 column 14
21.02 16:15:57 [Server] main/FATAL A critical error occurred
21.02 16:15:57 [Server] INFO net.fabricmc.loader.discovery.ModResolutionException: Could not find required mod: manhunt requires {manhunt-base @ [*]}
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.discovery.ModResolver.findCompatibleSet(ModResolver.java:180) ~[fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.discovery.ModResolver.resolve(ModResolver.java:787) ~[fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.FabricLoader.setup(FabricLoader.java:211) ~[fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.FabricLoader.load(FabricLoader.java:201) [fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:126) [fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:27) [fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_282]
21.02 16:15:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_282]
21.02 16:15:57 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_282]
21.02 16:15:57 [Server] INFO at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_282]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.launch.server.FabricServerLauncher.launch(FabricServerLauncher.java:62) [fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.launch.server.FabricServerLauncher.setup(FabricServerLauncher.java:106) [fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:50) [fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO Caused by: net.fabricmc.loader.util.sat4j.specs.ContradictionException: Creating Empty clause ?
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.util.sat4j.minisat.constraints.cnf.Clauses.propagationCheck(Clauses.java:117) ~[fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.util.sat4j.minisat.constraints.cnf.Clauses.sanityCheck(Clauses.java:97) ~[fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.util.sat4j.minisat.constraints.MixedDataStructureDanielWL.createClause(MixedDataStructureDanielWL.java:81) ~[fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.util.sat4j.minisat.core.Solver.addClause(Solver.java:401) ~[fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO at net.fabricmc.loader.discovery.ModResolver.findCompatibleSet(ModResolver.java:177) ~[fabric_1.16.5.jar:?]
21.02 16:15:57 [Server] INFO ... 12 more
21.02 16:15:57 [Multicraft] Server shut down (starting)
21.02 16:15:57 [Multicraft] Restarting crashed server in 300 seconds
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Fabric Loader Version: 1.16.5  (fabric_1.16.5.jar provided by Apex)
 - Mod Version: Latest, from Github = 2.0.2
 - Fabric API Version: Latest from Curseforge = 0.30.3+ 1.16

**Comment by Pepster3** [comment]
Not sure what I'm missing.  I had similar issues with the local version, but ended up getting Mod to run without ModMenu.  Looking to move to a hosted server for better performance.

**Comment by YTG1234** [comment]
Install [FLK](https://github.com/FabricMC/fabric-language-kotlin)

**Comment by Pepster3** [comment]
Holy moly that was fast!

Downloaded FLK, and added to MOD folder - server starts OK. 

Suggest adding that to the install instructions on Curseforge.     

Thanks!

**Comment by YTG1234** [comment]
> Suggest adding that to the install instructions on Curseforge.

Seeing as it is *above* the install instructions, I would have expected it to be seen. ![image](https://i.imgur.com/0oB23hz.png)

**Comment by Pepster3** [comment]
Yes, it was a bit of a forehead slapper, but bear in mind - You as a mod
author and experienced user know the significance of those badges, and can
adjust accordingly.  I scrolled down to "Instructions" and followed them to
the letter, figuring that they would include anything necessary.

I just installed Fabric, specifically to use this mod with my kids, so
there was a LOT to see and parse on that page to figure out how to get this
working.  Even more so for someone who does not have the experience to know
what things are critical and what are related to other topics (Like
GeyserMC).   I only added that suggestion to highlight that the
instructions are missing a critical element, which is highlighted above but
may not remain on the screen by the time a user has scrolled down that far.

All of this is nitpicky, and you have better things to do with your time
than deal with the OCD of some random schmuck.  Your mod works great now
and the kids are THRILLED with it, so *many many thanks* for putting this
together and sharing it.  And even more thanks for being so incredibly
responsive!

You rock,
- Pete
